App.tsx

```diff
@@ -74,7 +74,7 @@ class App extends React.PureComponent {
   render() {
     return (
       <>
-        {!__DEV__ && <WithCodePush />}
+        {!__DEV__ ? <WithCodePush /> : null}
```

Comment:
In this case I propose not to use negation
`{__DEV__ ? null : <WithCodePush /> }`